docs/user/guides/canonical-urls.rst

```diff
@@ -0,0 +1,32 @@
+How to Enable Canonical URLs
+============================
+
+In this guide, we introduce relevant settings for enabling canonical URLs in popular documentation frameworks.
+
+.. seealso::
```

Comment:
A seealso callout feels "heavy" here. I think we can just include it in the opening copy.